build_modules/pubspec.yaml

```diff
@@ -1,11 +1,11 @@
 name: build_modules
-version: 1.0.7+1
+version: 1.0.7+2
 description: Builders for Dart modules
 author: Dart Team <misc@dartlang.org>
 homepage: https://github.com/dart-lang/build/tree/master/build_modules
 
 environment:
-  sdk: "2.1.0"
+  sdk: ">=2.1.1-dev <3.0.0"
```

Comment:
do we know which dev release specifically introduced the change?

Reply:
Yes. The first dev release after 2.1.0. See https://github.com/dart-lang/sdk/commit/b484aada210c9d47fafdbe1809e8a466ead45ec9#diff-a56982f5d01aac6ee40c43140a933a0c